app/serializers/isa_tag_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ISATagSerializer < ActiveModel::Serializer
```

Comment:
I think this needs to be based off `BaseSerializer` or `SimpleBaseSerializer` to get the jsonapi and meta blocks